cmd/fyne/package.go

```diff
@@ -389,5 +393,5 @@ func (p *packager) doPackage() error {
 }
 
 func (p *packager) isMobile() bool {
-	return p.os == "ios" || p.os == "android"
+	return p.os == "ios" || p.os[0:7] == "android"
```

Comment:
Given that a typo in "ios" would cause this to panic I'd say please check the length of p.os before slicing at 7

Reply:
Agreed.  Any suggestions on the pending CI below?  I can't seem to get it to change to yellow.

Reply:
I'll have to address this later today since I'm busy for the rest of today.